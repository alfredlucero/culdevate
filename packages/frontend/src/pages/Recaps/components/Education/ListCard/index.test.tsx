import React from "react";
import { render } from "@testing-library/react";
import EducationListCard from "./index";

describe("<EducationListCard />", () => {
  test("should render without error", () => {
    const { container } = render(<EducationListCard onClick={() => {}} className="extra-list-class" testId="testId" />);

    expect(container).toMatchSnapshot();
  });
});
