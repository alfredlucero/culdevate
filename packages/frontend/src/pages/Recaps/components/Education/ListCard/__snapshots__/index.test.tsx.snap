// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EducationListCard /> should render without error 1`] = `
<div>
  <div
    class="bg-white shadow-md rounded px-8 py-6 w-64 extra-list-class"
    data-testid="testId"
  >
    <div
      class="mb-6 flex justify-center"
    >
      <div
        class="flex justify-center items-center p-3 rounded-lg bg-gray-200 recap-icon"
      >
        <div
          class="font-sans text-3xl text-teal-400"
          id=""
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-graduation-cap fa-w-20 "
            data-icon="graduation-cap"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 640 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M622.34 153.2L343.4 67.5c-15.2-4.67-31.6-4.67-46.79 0L17.66 153.2c-23.54 7.23-23.54 38.36 0 45.59l48.63 14.94c-10.67 13.19-17.23 29.28-17.88 46.9C38.78 266.15 32 276.11 32 288c0 10.78 5.68 19.85 13.86 25.65L20.33 428.53C18.11 438.52 25.71 448 35.94 448h56.11c10.24 0 17.84-9.48 15.62-19.47L82.14 313.65C90.32 307.85 96 298.78 96 288c0-11.57-6.47-21.25-15.66-26.87.76-15.02 8.44-28.3 20.69-36.72L296.6 284.5c9.06 2.78 26.44 6.25 46.79 0l278.95-85.7c23.55-7.24 23.55-38.36 0-45.6zM352.79 315.09c-28.53 8.76-52.84 3.92-65.59 0l-145.02-44.55L128 384c0 35.35 85.96 64 192 64s192-28.65 192-64l-14.18-113.47-145.03 44.56z"
              fill="currentColor"
            />
          </svg>
        </div>
      </div>
    </div>
    <h4
      class="font-mono font-bold text-gray-800 text-xl mb-4 text-center"
    >
      Education
    </h4>
    <div
      class="flex justify-center mb-4"
    >
      <button
        class="font-bold py-2 px-4 rounded bg-transparent text-teal-700 border border-teal-500 hover:bg-teal-500 hover:text-white hover:border-transparent mr-2"
        type="button"
      >
        <div
          class="flex items-center"
        >
          View
        </div>
      </button>
      <button
        class="font-bold py-2 px-4 rounded font-bold py-2 px-4 rounded bg-teal-500 text-white hover:bg-teal-700 "
        type="button"
      >
        <div
          class="flex items-center"
        >
          Add
        </div>
      </button>
    </div>
    <div
      class="flex justify-end"
    >
      <div
        class="rounded bg-gray-200 text-gray-700 text-sm p-2 w-8 flex justify-center items-center"
      >
        10
      </div>
    </div>
  </div>
</div>
`;
